build.fsx

```diff
@@ -34,6 +46,19 @@ let runDotNet cmd workingDir =
         DotNet.exec (DotNet.Options.withWorkingDirectory workingDir) cmd ""
     if result.ExitCode <> 0 then failwithf "'dotnet %s' failed in %s" cmd workingDir
 
+let serverWatcher() = 
+    async {
+        runDotNet "watch run" serverPath
+    }
+
+let clientWatcher openBrowser = 
+    async {
+        let openFlag = if openBrowser then "--open" else ""
```

Comment:
Sadly enough, this actually breaks everything. Have just spent an hour with that 🤦 

In case of false, the command has extra space at the end, and the `yarn` function interprets that as another parameter, making
`> "C:\Program Files (x86)\Yarn\bin\yarn.cmd" webpack-dev-server --config "C:\Users\peter\Desktop\CheckYourCzech\src\Client\webpack.development.js" ""` 
instead of 
`> "C:\Program Files (x86)\Yarn\bin\yarn.cmd" webpack-dev-server --config "C:\Users\peter\Desktop\CheckYourCzech\src\Client\webpack.development.js"`
and then failing with errors...

So yeah, that should be at least trimmed but better refactored or you can leave initial copypaste if you wish, just make sure things still work :D

Reply:
how could I reproduce that one?

Reply:
done